collections/made-in-vietnam/index.md

```diff
@@ -0,0 +1,37 @@
+---
+items:
+ - mlbvn
```

Comment:
```suggestion
```

Unfortunately, as per our contribution guidelines, we don't allow self promotion if you are a maintainer of the project:

> Maintainers of a project cannot add a topic or collection for their own project. If a topic is popular enough to warrant inclusion, someone else will add or improve it.

We do this so that project get curated if there is enough support/activity in the community. Thank you!